common/persistence/sql/blob.go

```diff
@@ -231,13 +231,13 @@ func taskListInfoFromBlob(b []byte, proto string) (*sqlblobs.TaskListInfo, error
 	return result, thriftRWDecode(b, proto, result)
 }
 
-func transferTaskInfoToBlob(info *sqlblobs.TransferTaskInfo) (p.DataBlob, error) {
-	return thriftRWEncode(info)
+func TransferTaskInfoToBlob(info *persistenceblobs.TransferTaskInfo) (p.DataBlob, error) {
+	return protoRWEncode(info)
```

Comment:
`RW` means read/write. Why do we have it here, as part of a func name?

Reply:
🤷‍♂ Just a quick rename, will change this when I move/cleanup blob.go